README.md

```diff
@@ -3,32 +3,29 @@
 
 # Playbook Design System
 
-Playbook is the first design system built for both Rails & React interfaces. Inspired by [Velocity](https://www.invisionapp.com/inside-design/design-resources/design-system-dashboard-ui-kit/), Playbook takes a modern design approach, and applies it in a way that makes it easy to support bleeding edge, or legacy systems. Playbook is built & maintained by the User Experience & Development teams at Power Home Remodeling, the largest home remodeler in the US.
+Playbook is the first design system built for both Rails & React interfaces. Inspired by [Velocity](https://www.invisionapp.com/inside-design/design-resources/design-system-dashboard-ui-kit/), Playbook takes a modern design approach and applies it in a way that makes it easy to support bleeding edge or legacy systems. Playbook is built & maintained by the User Experience & Development teams at [Power Home Remodeling](https://www.techatpower.com/), the largest home remodeler in the US.
 
-## Requirements
+## Development
 
-- Docker 17.09 or later
-- Docker Compose 1.17.1 or later
+### Requirements
 
-## Getting started
+- [asdf](https://github.com/asdf-vm/asdf)
+- Install language tools: `asdf install`
 
-1. run `yarn && bundle`
-1. Install overcommit hooks `bin/overcommit`
-1. run `bin/rails s`
-1. run `./bin/webpack-dev-server`
-3. open [http://localhost:3000](http://localhost:3000)
+### Getting started
 
-<details><summary>Making changes to the Gemfile:</summary>
-<p>
+1. Install commit hooks `yarn prepare`
+1. Install all projects dependencies: `yarn install`
+1. Install website ruby dependencies: `cd playbook-website && bundle`
```

Comment:
The location these commands are run is a bit strange, as they jump around a bit. For example, here I need to be inside the website folder. But to run the website I would need to be in root. Not sure if anything can be done about that in this version. Perhaps we just need to make the readme a bit more clear about location as I ran `yarn start-dev` from the wrong folder.

Reply:
```
➜  playbook-website git:(decouple-website) bundle
Ignoring nokogiri-1.10.9 because its extensions are not built. Try: gem pristine nokogiri --version 1.10.9
Ignoring redcarpet-3.4.0 because its extensions are not built. Try: gem pristine redcarpet --version 3.4.0
Warning: the running version of Bundler (1.17.2) is older than the version that created the lockfile (1.17.3). We suggest you upgrade to the latest version of Bundler by running `gem install bundler`.
Your Ruby version is 2.6.3, but your Gemfile specified 2.5.0
```

I ran `bundle` as described above and this is what I received. 

```
➜  playbook-website git:(decouple-website) ruby -v
ruby 2.5.0p0 (2017-12-25 revision 61468) [x86_64-darwin20]
```

Reply:
Fixed by running `gem install bundler -v 1.17.3` so that might be good to add to the docs @xjunior